compiler/docgen.nim

```diff
@@ -819,6 +821,18 @@ proc complexName(k: TSymKind, n: PNode, baseName: string): string =
       result.add(defaultParamSeparator)
       result.add(params)
 
+proc rstLinkName(k: TSymKind, n: PNode, baseName: string): string =
+  ## Creates a link anchor in the format compatible with the corresponding
+  ## human-readable RST references.
+  result = baseName
```

Comment:
can you add tests with comma separated links, eg:
```nim
## see fn1_, fn2_, `fn3`_, `fn4`_
```

Reply:
done.